package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Converts a JSON document containing the asset_tracker_v2 device and sensor data encoded as LwM2M to the JSON document required by nRF Asset Tracker.",
   "main": "index.js",
   "scripts": {
-    "test": "find ./ -type f -name '*.spec.ts' ! -path '*/node_modules/*' -exec npx tsx {} ';'",
+    "test": "npx tsx --no-warnings --test --test-reporter spec ./src/**/*.spec.ts",
```

Comment:
`src/converter.spec.ts` tests are being ignored by `./src/**/*.spec.ts`. 

I tried with `npx tsx --no-warnings --test --test-reporter spec ./src/*.spec.ts` but the rest of the tests are being ignored then. Also, I tried to use the && command but it didn't work.

Reply:
Fixed.